applications/jupyter-lab-hpc-native/app.json

```diff
@@ -0,0 +1,82 @@
+{
+    "id": "jupyter-lab-hpc-native",
+    "version": "1.0.2",
+    "description": "Run an interactive Jupyter Notebook session with optional MPI support.",
+    "owner": "${apiUserId}",
+    "enabled": true,
+    "runtime": "ZIP",
+    "containerImage": "tapis://cloud.data/corral/tacc/aci/CEP/applications/v3/jupyter-lab-hpc-native/jupyter-lab-hpc-native.zip",
+    "jobType": "BATCH",
+    "maxJobs": -1,
+    "maxJobsPerUser": -1,
+    "strictFileInputs": true,
+    "jobAttributes": {
+        "description": null,
+        "dynamicExecSystem": false,
+        "execSystemId": "frontera",
+        "execSystemExecDir": "${JobWorkingDir}",
+        "execSystemOutputDir": "${JobWorkingDir}/output",
+        "execSystemLogicalQueue": "development",
```

Comment:
default queue can be rtx-dev, with updated default cores per node and memory per node below

Reply:
Changed it to "rtx-dev" and add:
        "coresPerNode": 16,
        "memoryMB": 128000,